repmgr-17.yaml

```
@@ -216,11 +211,10 @@ test:
   environment:
     contents:
       packages:
-        - postgresql-16
+        - repmgr
```

```suggestion
        - repmgr-17
```
should we also pull in specific `repmgr-17` here for testing? I was mostly thinking on line that if we copy paste then this will always pull in the latest?

---

`repmgr-17` package already provides `repmgr` but let's omit it, since `test:` pipeline fetches the _current_ package already. Thanks.